### ISSUE TITLE
Safari needs babel-preset-stage-2

### ISSUE THREAD

**Issue by bhj** [GlBj/MyWebPack#3]
Hi, with v1.2.0 of this repo (v1.1.1 works), desktop and mobile Safari will throw

`SyntaxError: Unexpected token '...'. Expected a property name.`

This is related to https://github.com/twbs/bootstrap/issues/20709

I wasn't sure of your preferred way of fixing this, but might it make sense to ditch `.babelrc` and keep that stuff in the webpack config just so it's all in one place? I like the way this repo is organized; thanks!

**Comment by bhj** [comment]
Another issue with Safari (and possibly Firefox, let me know if you want me to open another issue):

https://github.com/tcoopman/image-webpack-loader/issues/111

Still appears to be a problem, even with `image-webpack-loader@4.2.0`. Disabling the loader solves it.

**Comment by GlBj** [comment]
You are completely right! I only tested it through chrome. Hmmm, now I have to figure out to how fix it in a smart way.

I tested in: chrome, firefox, edge and safari and right now it only loads the background in chrome.

I leave this issue open until i found a solution that works on most webbrowsers.

**Comment by bhj** [comment]
@GlBj Hey, after adding the stage-2 preset to "fix" Safari:

.babelrc
```
{
    "presets": ["env","stage-2","minify"]
}
```

webpack.config.js
```
...
  module: {
    rules: [
      {
        test: /\.js$/,
        include: [
          __dirname + "/js/",
          __dirname + "/node_modules/bootstrap/js/"
        ],
        loader: "babel-loader"
      },
...
```

I just noticed that trying to collapse the nav after expanding it throws (in all browsers)

`TypeError: undefined is not an object (evaluating 'i.HIDE')`

Removing the minify plugin from `.babelrc` fixes it, so I guess we need separate rules for Bootstrap and non-Bootstrap js? Or is there a better way? Thanks again.

**Comment by GlBj** [comment]
@bhj :+1: I tested some other solutions, but no one was as good as yours. Thanks!

**Comment by bhj** [comment]
@GlBj I think there is still that issue with the nav dropdown - make Safari narrow enough so that the nav collapses, then try toggling the expand/collapse to get the error on collapse. I'm not sure why `stage-2` and `minify` preset don't play well together with the Bootstrap stuff...

One other (unrelated) thing I noticed, but easy to fix, it looks like you are defining the `providePluginList` but not passing it to Webpack yet.

**Comment by GlBj** [comment]
The problem is I'm on the dark side and use linux, windows and android. If U have a solution that works for everyone please push it.

Regards,
Glenn

**Comment by GlBj** [comment]
@bhj I believe the bug is gone now?

- I found out that .babelrc didn't load in babel-loader so i added it in the webpack.config.js instead.
- I also got back to load the full package of bootstrap, instead of loading the seperated modules. That solved the issue with dropdown didn't work in some browsers. Not realy sure why?

Let me know if it works on Safari to, so I can close this bug? If [sajt.online](https://sajt.online/) loads alright in Safari I know it works?

Regards,
Glenn
